var/spack/repos/builtin/packages/etsf-io/package.py

```diff
@@ -32,6 +32,13 @@ class EtsfIo(Package):
     patch("tests_module.patch")
     patch("tests_init.patch")
 
+    def configure_args(self):
+        args = [
+            "FFLAGS={0}".format(self.compiler.f77_pic_flag),
+            "FCFLAGS={0}".format(self.compiler.fc_pic_flag),
+        ]
+        return args
```

Comment:
Can we define a `flag_handler` instead?

Reply:
How to do that for `FCFLAGS`? (i.e. Fortran 90) As far as I can see the `flag_handler` only supports `fflags` (Fortran 77).